eslint/config.js

```diff
@@ -0,0 +1,171 @@
+module.exports = {
+	parser: 'babel-eslint',
+	extends: [
+		'wordpress',
+		'plugin:wordpress/esnext',
+		'plugin:react/recommended',
+		'plugin:jsx-a11y/recommended',
+	],
+	env: {
+		browser: false,
```

Comment:
I think this is false because this is covering non-built linting, in other words what is being linted is NOT what gets served to the browser (we only serve _built_ files to the browser).